tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/hdfs/WebHdfsClient.java

```diff
@@ -183,9 +183,7 @@ public void loadFile(String path, OutputStream outputStream)
         }
     }
 
-    @Override
-    @SuppressWarnings("unchecked")
-    public long getLength(String path)
+    private Object getAttributeValue(String path, String attribute)

```

Comment:
I think it will be hard to implement this method with using regular HDFS client instead of WebHDFS. We are going to introduce such implementation, because on Facebook clusters WebHDFS is not exposed. So, i would rather keep it private.